src/services/FilterService.js

```diff
@@ -75,7 +78,8 @@ export const getInitialFilter = (allNeighborhoods) => {
     const initialFilter = {
```

Comment:
not part of this PR but can we simply return this instead of declaring this variable?

Reply:
I think so? I've added it to the quip doc to look at later.